### ISSUE TITLE
Better UI design scheme for Verse card

### ISSUE THREAD

**Issue by redraushan** [gita/BhagavadGita#248]
# What

- Elements of the Verse card should be more aligned.
- Social sites liks - FB, Twitter, and share should be shown at the bottom of the card, giving more clarity to the Mantra heading.
- Unified design for the Buttons.
- Better contrast ratio between background color and font-color of the verse.

# How
I have created a POC that that demonstrates the above mentioned changes, also I am adding the current version of the Verse card. Please compare both and and let me know any changes you would like to make to make it more better.

# Current Verse Card
![old-verse](https://user-images.githubusercontent.com/12084473/53295423-a1689d80-3814-11e9-9395-8c6893bcfd08.png)

# POC Verse Card
![screenshot-bhagavadgita io-2019 02 24-09-00-40](https://user-images.githubusercontent.com/12084473/53295425-a9c0d880-3814-11e9-89f4-7c1bc9dbb1d8.png)

**Comment by redraushan** [comment]
# Question
- Can we have a replacement of samarkan font? To me it's a bit more decorative and less readable to the eyes unless you put a bit effort to read.

![font](https://user-images.githubusercontent.com/12084473/53295518-53549980-3816-11e9-9ba2-86b214f25693.png)

**Comment by samanyougarg** [comment]
Hi @redraushan,

> * Elements of the Verse card should be more aligned.

Agree with you on this.

> * Social sites like - FB, Twitter, and share should be shown at the bottom of the card, giving more clarity to the Mantra heading.

Yes, you are right. The share buttons do obstruct the reading.

> * Unified design for the Buttons.

Not too sure about this. I am following the material design guidelines as implemented in MDBootstrap. Happy to discuss more about this.

> * Better contrast ratio between the background color and font-color of the verse.

Yes, you are right, but I think black is the most readable color when it comes to text. Dark brown makes it a bit harder to read. Again, happy to discuss more about this.

**Comment by samanyougarg** [comment]
> # Question
> * Can we have a replacement of samarkan font? To me it's a bit more decorative and less readable to the eyes unless you put a bit effort to read.
> 
> ![font](https://user-images.githubusercontent.com/12084473/53295518-53549980-3816-11e9-9ba2-86b214f25693.png)

Yes, that's right. I basically wanted to distinguish the chapter name in Sanskrit from that in English. Samarkan was the only font that was similar to Sanskrit that I found. Do you know of any alternative fonts?

**Comment by redraushan** [comment]
> Yes, that's right. I basically wanted to distinguish the chapter name in Sanskrit from that in English. Samarkan was the only font that was similar to Sanskrit that I found. Do you know of any alternative fonts?

I am yet to find one, but for the time being can we go with the rest part? Let me know when you are available, we can have a call to conclude the same.

Thanks

**Comment by vishwasnavadak** [comment]
> > # Question
> > 
> > * Can we have a replacement of samarkan font? To me it's a bit more decorative and less readable to the eyes unless you put a bit effort to read.
> > 
> > ![font](https://user-images.githubusercontent.com/12084473/53295518-53549980-3816-11e9-9ba2-86b214f25693.png)
> 
> Yes, that's right. I basically wanted to distinguish the chapter name in Sanskrit from that in English. Samarkan was the only font that was similar to Sanskrit that I found. Do you know of any alternative fonts?

Since that font is just used for decoration, can we just get it changed for more readability?

PS: Great site!

**Comment by redraushan** [comment]
@vishwasnavadak Do you have any font recommendations that we can take as a replacement for with the current one? 

I guess it should be less decorative and should keep the flavour of Sanskrita.

**Comment by vishwasnavadak** [comment]
@redraushan I can search for some. But as for as sanskrita is considered, I think Samarkan is the only font available. 
I will get back with some fonts.

**Comment by samanyougarg** [comment]
Hi @redraushan and @vishwasnavadak,

Sorry for the late reply.

I found another font called "Ananda Neptouch". Please have a look at it and let me know if you think this one is more readable.

Samarkan - 
<img width="342" alt="Screenshot 2019-06-15 at 16 20 53" src="https://user-images.githubusercontent.com/32533774/59550557-971c0c00-8f89-11e9-9760-6fdfbc08bff3.png">

Ananda Neptouch - 
<img width="341" alt="Screenshot 2019-06-15 at 16 17 27" src="https://user-images.githubusercontent.com/32533774/59550552-91262b00-8f89-11e9-91be-b950630d599e.png">

**Comment by redraushan** [comment]
Hi @samanyougarg ,

What a great font you have come up with :+1: 
I would like to see it implemented. @vishwasnavadak  what do you think?

**Comment by vishwasnavadak** [comment]
Hey, great that you got a replacement font. It is way more readable without the top line. 

Go for it.

**Comment by vishwasnavadak** [comment]
Let me know if you need any help in putting things together

**Comment by redraushan** [comment]
Hi @vishwasnavadak,

I will come up with the PR acccumulating all the changes that I have proposed above in the thread along with new font :100: 

Thanks
@samanyougarg

**Comment by samanyougarg** [comment]
Awesome!
@redraushan please let me know when you submit the PR.
